### ISSUE TITLE
Option for exit on SIGINT

### ISSUE THREAD

**Issue by crislin2046** [GoogleChrome/chrome-launcher#183]
This project has saved me a lot of time instead of writing it myself, really grateful for that!!! Thanks 😃 

It would be nice to switch off the `process.exit(...)` behavior that occurs on SIGINT.

What if `chrome-launcher` is embedded by another program that has its own handler on SIGINT? In that case chrome-launcher's handler might kill the node process before the other program's SIGINT handler has completed.

You *can* have async actions in a SIGINT handler. And I don't like the workaround of spawning a new node process just to run chrome launcher. 

I think a switch like, 

```js
   sigintDefaultAction: false, // or
   killProcess: false, // or
   killProcessOnSigint: false, // you get the idea
```

**Comment by crislin2046** [comment]
Closed in #184

**Comment by crislin2046** [comment]
Closed because idea moves the problem around, not solves it.
